nipy/algorithms/clustering/imm.py

```diff
@@ -234,7 +236,7 @@ def cross_validated_update(self, x, z, plike, kfold=10):
         if np.isscalar(kfold):
             aux = np.argsort(np.random.rand(n_samples))
             idx = - np.ones(n_samples).astype(np.int)
-            j = np.ceil(n_samples / kfold)
+            j = int(math.ceil(n_samples / kfold))
```

Comment:
All right. In that case, we don't really care about the exact value, so both are fine.